content/reference/admin/private_locations/configuration/azure/index.md

```diff
@@ -26,7 +26,22 @@ Check Azure documentation pages to find these values:
 * [Client id and secret](https://learn.microsoft.com/en-us/answers/questions/834401/hi-i-want-my-client-id-and-client-secret-key)
 * [Subscription id](https://learn.microsoft.com/en-us/azure/azure-portal/get-subscription-tenant-id)
 
-### Control plane configuration file
+## System requirements
+
+Azure private locations rely on some dependencies.
+
+So when using a custom image, make sure following are available:
+
+- [cloud-init](https://learn.microsoft.com/en-us/azure/virtual-machines/custom-data) integration.
+- [jq](https://jqlang.github.io/jq/download/) a lightweight and flexible command-line JSON processor.
+- [curl](https://curl.se/download.html) a command line tool and library for transferring data with URLs
+- [Java runtime environment](https://openjdk.org/install/): OpenJDK 64bits LTS versions: 11, 17 or 21 (see [Gatling prerequisites](https://gatling.io/docs/gatling/tutorials/installation/#java-version))
+
+{{< alert tip >}}
+Learn how to tune the OS for more performance, configure the open files limit, the kernel and the network [here](https://gatling.io/docs/gatling/reference/current/core/operations/).
```

Comment:
I usually avoid using "here" for a link in the documentation. You could replace "here" with "in the Documentation" and add the link to the word Documentation.